### DISCUSSION TITLE
我也写了一个值得买的签到和任务，但是遇到了2个问题。请教是怎么解决的。

### DISCUSSION THREAD

**Discussion by lilingkong** [blackmatrix7/ios_rule_script#866] [Q&A]
1.签到绕过验证码。（当前已经解决）
2.收藏商品、文章等，只要用脚本跑的，账号就会掉线（用的web接口，从2月13号开始掉线。猜测以该时间线为准，张大妈开发修改了规则），但奇怪的是，我登录账号人工点收藏就没事。 他是怎么识别人工和机器的？
作者有微信吗？这里留个联系方式，交流下怎么批量执行任务不掉线，我这里有大量的值友遇到这个头疼的问题。 
v： linken_li 
thx !!!

**Comment by blackmatrix7** [comment]
你可以留下一个电报号码，我来加你。

我也想交流下是如何绕过验证码的。

另外点收藏和文章，我没有做过什么特殊处理，只是增加了一些时间间隔， 之前测试过太短的事件间隔会导致账号异常。

脚本都是开源的，你可以看下源代码能否帮上你。
